doc/boot-partition.md

```diff
@@ -130,21 +134,34 @@ i.e. valid for all architectures (s390, see below)
 - order: /boot, swap, /
 - ext4
 
-x86-legacy
-----------
-- grub boot.img (stage 1) in MBR
-- install core.img (stage 1.5) in free space between partition table and 1. partition
-- if there isn't enough free space -> create additional partition
-- install all grub parts on same disk
-
-x86-efi
--------
-if system efi partition is already there -> reuse it
-if not create:
-- /boot/efi
-- fat32
-- size: 33 MB, 200MB
-- limit: below 2TB
+## x86-legacy
+
+*[this assumes grub will be installed into mbr]*
+
+- required size for grub embedding (currently): 84 kB; to be safe, check for >= 256 kB
```

Comment:
The size of core.img would be bigger if boot partition is on mdadm and lvm. But I think 256kB is sufficient for that also.